app/views/expenses/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= simple_form_for expense do |f| %>
+  <%= f.input :name, placeholder: "Your name" %>
+  <%= f.input :due_date, order: [:day, :month, :year] %>
+  <%= f.input :frequency, placeholder: "weekly, fortnightly, monthly" %>
```

Comment:
Translations ^. You might be able to just call `Expense.frequencies`??

Reply:
Would have been nice, but I couldn't get it to work. 
Done in sha: 740613b